crates/astria-proto/src/native/sequencer/v1alpha1.rs

```diff
@@ -3,18 +3,721 @@ use std::{
     fmt::Display,
 };
 
-use crate::generated::sequencer::v1alpha1;
+use ed25519_consensus::{
```

Comment:
is there a reason to have all of these types in 1 file? it's rather long lol

Reply:
Do you mean `ed25519_consensus` or this module in general? It's not so uncommon to have relatively large rust files as long as it makes sense that they are grouped together, which I think is the case here.

Reply:
sorry yeah, i meant the whole file. okay, makes sense